cmd/provisioner-localpv/app/start.go

```diff
@@ -110,7 +112,14 @@ func Start(cmd *cobra.Command) error {
 	)
 	klog.V(4).Info("Provisioner started")
 	//Run the provisioner till a shutdown signal is received.
-	pc.Run(ctx)
+	go pc.Run(ctx)
+
+	if menv.Truthy(menv.OpenEBSEnableAnalytics) {
+		analytics.New().Build().InstallBuilder(true).Send()
+		go analytics.PingCheck()
+	}
+
+	<-ctx.Done()
```

Comment:
Instead of creating a separate goroutine for pc.Run(ctx), could you instead just block the main goroutine with the call?
You can call pc.Run(ctx) after the analytics 'if' block.